internal-packages/workflow-designer-ui/src/editor/properties-panel/file-node-properties-panel/index.tsx

```diff
@@ -1,26 +1,44 @@
-import type { FileNode } from "@giselle-sdk/data-type";
+import type { FileCategory, FileNode } from "@giselle-sdk/data-type";
 import { useWorkflowDesigner } from "giselle-sdk/react";
-import { PdfFileIcon } from "../../../icons";
+import { FileNodeIcon } from "../../../icons/node";
 import {
 	PropertiesPanelContent,
 	PropertiesPanelHeader,
 	PropertiesPanelRoot,
 } from "../ui";
-import { FilePanel } from "./file-panel";
+import { FilePanel, type FileTypeConfig } from "./file-panel";
+
+const fileType: Record<FileCategory, FileTypeConfig> = {
+	pdf: {
+		accept: ["application/pdf"],
+		label: "PDF",
+	},
+	text: {
+		accept: ["text/plain", "text/markdown"],
```

Comment:
nits: 

It seems "text/markdown" accepted, but I cannot upload the [sample.md](https://github.com/user-attachments/files/19121329/sample.md).

<img width="351" alt="image" src="https://github.com/user-attachments/assets/d0de529b-250d-40a2-9072-2a752103e777" />

Reply:
New issue
- https://github.com/giselles-ai/giselle/issues/428